bloom-instance/network/inputs.tf

```diff
@@ -0,0 +1,19 @@
+variable "subnet_map" {
+  type        = object({
+    public = list(object({
+      az   = string
+      cidr = string
+    }))
+
+    backend = list(object({
+      az   = string
+      cidr = string
+    }))
+
+    data = list(object({
+      az   = string
+      cidr = string
+    }))
+  })
+  description = "The subnets to create in our VPC"
+}
```

Comment:
I should have made this more clear in the issue, but we don't need to put network resources in a module, so just add the subnet_map variable to the vars.tf file in the root.